### PR TITLE
bump manifest for 4.3.0.  fixes #188

### LINKED ISSUE #188: olm data needs to be updated for 4.3
https://github.com/openshift/elasticsearch-operator/tree/release-4.3/manifests
file not found: manifests/4.3/image-references

### PR DESCRIPTION
**Pull request by jcantrill**
fixes #188 .  bump to 4.3

### CODE REVIEW

**Comment by jcantrill** [comment]
cc @richm @ewolinetz

File: manifests/4.3/elasticsearch-operator.v4.3.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ metadata:
     containerImage: quay.io/openshift/origin-elasticsearch-operator:latest
     createdAt: 2019-02-20T08:00:00Z
     support: AOS Cluster Logging, Jaeger
-    olm.skipRange: ">=4.1.0 <4.2.0"
+    olm.skipRange: ">=4.2.0 <4.3.0"
```
Comment (ewolinetz) [inline manifests/4.3/elasticsearch-operator.v4.3.0.clusterserviceversion.yaml]: shouldn't this be `>=4.1.0 <4.3.0`?

Comment (jcantrill) [reply manifests/4.3/elasticsearch-operator.v4.3.0.clusterserviceversion.yaml]: No.  This is intentional and fits our policy with prior releases to force stepwise upgrades

**Comment by bparees** [comment]
@jcantrill @richm this is failing semi regularly due to test timeouts....you may need to increase the cpu+memory resources assigned to the unit test job.

**Comment by bparees** [comment]
correct: to the e2e job

**Pull request merged**